apps/webapp/web/controllers/admin/page_controller.ex

```diff
@@ -17,6 +17,18 @@ defmodule Webapp.Admin.PageController do
     render_view(conn)
   end
 
+  def list_of_docs(conn, _params) do
```

Comment:
I think it would be better if this action is in a `DocsController`, outside `Admin`. Then the name `list_of_docs` would be just `index`. Of course, the `doc_title` helper fn can also be moved accordingly